md/single-sign-on-with-cas.md

```diff
@@ -136,9 +142,15 @@ Bonita BPM uses the CAS LoginModule in the JASIG implementation, so see the CAS
 2. In the `CasLoginModule` configuration, check that the `principalGroupName` property is set to `CallerPrincipal`.  
    This is required to retrieve the username from the Bonita application.  
    Bonita BPM uses the CAS LoginModule in the JASIG implementation, so see the CAS LoginModule section of the [Jasig documentation](https://wiki.jasig.org/display/CASC/JAAS+Integration) for more information.
-3. Copy `cas-client-core-3.3.1.jar` from `BonitaBPMSubscription-7.3.x-deploy` into the `/lib` directory.
-4. Copy `commons-logging-1.1.1.jar` from `BonitaBPMSubscription-7.3.x-deploy` into the `/lib` directory.
+3. Copy `cas-client-core-3.3.1.jar` from `BonitaBPMSubscription-7.4.x-deploy/cas-3.3.1-module/org/jasig/cas/main` into the `TOMCAT_HOME/server/lib` directory.
```

Comment:
same thing here: I would use x.x.x instead of specifying any version number as we don't update the version numbers in all the doc pages when creating a new version